pkg/crc/constants/constants_windows.go

```diff
@@ -12,3 +17,28 @@ const (
 )
 
 var TrayBinaryPath = filepath.Join(CrcBinDir, "tray-windows", TrayBinaryName)
+
+func GetHomeDir() string {
+	// if not running in interactive mode, it is the daemon service
+	// in that case we have the user's home directory as the third
+	// argument in the service executable invocation
+	isIntSess, err := svc.IsAnInteractiveSession()
+	if err == nil {
+		if !isIntSess && len(os.Args) > 2 {
+			return os.Args[2]
+		}
+	}
+
+	if homeDrive, homePath := os.Getenv("HOMEDRIVE"), os.Getenv("HOMEPATH"); len(homeDrive) > 0 && len(homePath) > 0 {
+		homeDir := filepath.Join(homeDrive, homePath)
```

Comment:
Is this required now that we use `os.UserHomeDir()`?

Reply:
Yes, this is to take care of situations when the `%USERPROFILE%` is not set in windows